src/supermarket/app/controllers/api/v1/quality_metrics_controller.rb

```diff
@@ -1,37 +1,36 @@
 class Api::V1::QualityMetricsController < Api::V1Controller
-  before_action :check_cookbook_name_present, except: :license_evaluation
-  before_action :check_authorization, except: :license_evaluation
-  before_action :find_cookbook_version, except: :license_evaluation
+  before_action :check_cookbook_name_present
+  before_action :check_authorization
+  before_action :find_cookbook_version
 
   #
-  # POST /api/v1/quality_metrics/foodcritic_evaluation
```

Comment:
Can we add a deprecation warning for anyone hitting this API?

Reply:
ok will make this change

Reply:
We do not see this api being consumed externally and we have completely removed the foodcritic logic. 
cc @skkprogress

Reply:
Looks like we never actually documented this one so ignore me. It's not something we have to worry about